mailpoet/lib/Mailer/Mailer.php

```diff
@@ -16,6 +18,12 @@ class Mailer {
   const METHOD_PHPMAIL = 'PHPMail';
   const METHOD_SMTP = 'SMTP';
 
+  public function __construct(
+    MailerMethod $mailerInstance
+  ) {
+    $this->mailerInstance = $mailerInstance;
```

Comment:
Oh, this confused me for a while — let's rename the variable and property to `mailerMethod` or just `method` :)